README.md

```diff
@@ -171,6 +171,28 @@ docker ps --format="table {{.ID}}\t{{.Image}}\t{{.Labels}}"
 ## Deployment
 
 This service is to be deployed to the Kubernetes cluster once it is merged.
+
+### Checking on which k8s you are before doing anything
+
+by running :
+
+```bash
+kubectl config get-contexts
```

Comment:
(for the lazy ones: `kubectx` would do the same with only one word to be typed :wink: )